app/views/works/index.html.erb

```diff
@@ -0,0 +1,79 @@
+<h3>List of Works</h3>
+
+<section class="works-table">
+  <h4>Albums</h4>
+  <table>
+    <thead>
```

Comment:
Again, this HTML pattern is repeated, and would make a good candidate for a partial.